.github/copilot-instructions.md

```diff
@@ -6,6 +6,10 @@ You are an expert Rust programmer. You write safe, efficient, maintainable, and
 -   Do not be overly apologetic and focus on clear guidance.
 -   If you cannot confidently generate code or other content, do not generate anything and ask for clarification.
 
+## Prerequisites
+
+-   To use MCP tools, users must have PowerShell installed. Provide [PowerShell installation instructions](https://learn.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.5) if not installed, and recommend restarting VSCode to start the MCP server.
```

Comment:
Which MCPs? Clearly not all. Seems like we should be concrete here. Are they Azure MCPs? Should we have a `.vscode/mcp.json` file checked in? This is too vague as is.

Reply:
In that case, you should say "Azure SDK MCPs" or whatever you call the internally-focused ones. Let's not confuse users since we also have the "Azure MCPs" that are more externally-focused.

/cc @maririos